package.json

```diff
@@ -1,38 +1,36 @@
 {
 	"name": "coder-remote",
-	"publisher": "coder",
 	"displayName": "Coder",
-	"description": "Open any workspace with a single click.",
-	"repository": "https://github.com/coder/vscode-coder",
 	"version": "1.9.1",
-	"engines": {
-		"vscode": "^1.73.0"
-	},
-	"license": "MIT",
+	"description": "Open any workspace with a single click.",
+	"categories": [
+		"Other"
+	],
 	"bugs": {
 		"url": "https://github.com/coder/vscode-coder/issues"
 	},
-	"icon": "media/logo.png",
-	"extensionKind": [
-		"ui"
-	],
-	"capabilities": {
-		"untrustedWorkspaces": {
-			"supported": true
-		}
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/coder/vscode-coder"
 	},
-	"categories": [
-		"Other"
-	],
-	"extensionPack": [
-		"ms-vscode-remote.remote-ssh"
-	],
-	"activationEvents": [
-		"onResolveRemoteAuthority:ssh-remote",
-		"onCommand:coder.connect",
-		"onUri"
-	],
+	"license": "MIT",
+	"publisher": "coder",
+	"type": "commonjs",
 	"main": "./dist/extension.js",
+	"scripts": {
+		"build": "webpack",
+		"fmt": "prettier --write .",
+		"lint": "eslint . --ext ts,md,json",
+		"lint:fix": "yarn lint --fix",
+		"package": "webpack --mode production --devtool hidden-source-map",
+		"package:prerelease": "npx vsce package --pre-release",
+		"pretest": "tsc -p . --outDir out && yarn run build && yarn run lint",
```

Comment:
Do we have to compile separately from Webpack because vscode-test requires that or something?  Is there a reason it cannot use the Webpack bundle?

Reply:
I'm not sure, I'll see if I can ditch it in my next PR 👍